### ISSUE TITLE
ReCAPTCHA couldn't find user-provided function

### ISSUE THREAD

**Issue by b-rhodes** [UndefinedOffset/silverstripe-nocaptcha#50]
```
ReCAPTCHA couldn't find user-provided function: Nocaptcha-QuoteForm_QuoteForm_Captcha
recaptcha__en.js:formatted:13193
```

I'm getting this javascript error when submitting a form, any idea what could be causing it?

**Comment by UndefinedOffset** [comment]
What version of the module are you using and what version of the cms are you using? Are there any other JavaScript errors prior to attempting to submit the form?

**Comment by b-rhodes** [comment]
Sorry for late reply, I'm using Silverstripe 4.1.2
There is another javascript error, but it's unrelated.

**Comment by UndefinedOffset** [comment]
Ok three more questions (sorry just trying to narrow down the issue):

1. What version of this module are you using
2. What version of jQuery are you using?
3. Are you using jQuery validator, if so which version of it?

**Comment by dunatron** [comment]
I am also getting this issue.
ReCAPTCHA couldn't find user-provided function: Nocaptcha-Form_AskAQuestion_Captcha

undefinedoffset/silverstripe-nocaptcha (2.0.3)
jQuery v: 1.10.2

**Comment by extravio** [comment]
I think the problem is that the "window.Nocaptcha-Form_[Whatever]_Captcha" function is only defined for invisible Recaptcha (NocaptchaField.js line 38).
However, the callback function is still set to this global function independently of the "data-size" attribute  (NocaptchaField.js line 57).

**Comment by UndefinedOffset** [comment]
I think @extravio  is correct, I'll see if I can get a fix done up later today

**Comment by UndefinedOffset** [comment]
Fixed and tagged as 2.0.4
